rpms/mysql-client.pan

```diff
@@ -1,14 +1,9 @@
 unique template rpms/mysql-client;
 
-include { 'rpms/group/core' };
-
-prefix '/software/packages';
-
-
-'{MySQL-python}' ?= nlist();
-'{mysql}' ?= nlist();
-'{mysql-connector-java}' ?= nlist();
-'{mysql-connector-odbc}' ?= nlist();
-'{perl-DBD-MySQL}' ?= nlist();
-'{perl-DBI}' ?= nlist();
-'{sqlite}' ?= nlist();
+'/software/packages' = pkg_repl('MySQL-python');
```

Comment:
Fair enough, I don't think it's an important different on these largely static templates though.
I asked because it obscured the actual change (which was switch MySQL to MariaDB).